merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -122,6 +122,14 @@ export class Windows {
   /**
    * Invert all the windows produced this.
    */
+  public and(...windows: Windows[]): Windows {
+    return Windows.And(this, ...windows);
+  }
+
+  public or(...windows: Windows[]): Windows {
+    return Windows.Or(this, ...windows);
+  }
+   */
```

Comment:
Extra */

Reply:
same as above, this is fixed in commit 3